synapse/storage/databases/main/room.py

```diff
@@ -1411,6 +1411,54 @@ async def add_event_report(
             desc="add_event_report",
         )
 
+    async def get_event_report(self, report_id: int) -> Dict[str, Any]:
+        """Retrieve a an event report
+
+        Args:
+            report_id: ID of reported event in database
+        Returns:
+            event_report: json list of information from event report
+        """
+
+        def _get_event_report_txn(txn, report_id):
+
+            sql = """
+                SELECT
+                    er.id,
+                    er.received_ts,
+                    er.room_id,
+                    er.event_id,
+                    er.user_id,
+                    er.reason,
+                    er.content,
+                    events.sender,
+                    room_aliases.room_alias,
+                    event_json.json AS event_json
+                FROM event_reports AS er
+                LEFT JOIN room_aliases
+                    ON room_aliases.room_id = er.room_id
+                JOIN events
+                    ON events.event_id = er.event_id
+                JOIN event_json
+                    ON event_json.event_id = er.event_id
+                WHERE er.id = ?
+            """
+
+            txn.execute(sql, [report_id])
+            event_report = self.db_pool.cursor_to_dict(txn)
```

Comment:
```suggestion
            rows = self.db_pool.cursor_to_dict(txn)
            
            if not rows:
                return None
            
            event_report = rows[0]
```

`cursor_to_dict` actually returns a List of dictionaries.